pkg/bbgo/order_executor_general.go

```diff
@@ -222,7 +219,9 @@ type OpenPositionOptions struct {
 	Tags  []string         `json:"-" yaml:"-"`
 }
 
-func (e *GeneralOrderExecutor) OpenPosition(ctx context.Context, options OpenPositionOptions) error {
+var Delta fixedpoint.Value = fixedpoint.NewFromFloat(0.005)
```

Comment:
rename to quantityReduceDelta? and add a doc comment for this

Reply:
done